administration/eslint.config.mjs

```diff
@@ -0,0 +1,219 @@
+import { fixupConfigRules, fixupPluginRules } from '@eslint/compat'
+import { FlatCompat } from '@eslint/eslintrc'
+import typescriptEslint from '@typescript-eslint/eslint-plugin'
+import tsParser from '@typescript-eslint/parser'
+import jest from 'eslint-plugin-jest'
+import jsxA11Y from 'eslint-plugin-jsx-a11y'
+import jsxExpressions from 'eslint-plugin-jsx-expressions'
+import preferArrow from 'eslint-plugin-prefer-arrow'
+import react from 'eslint-plugin-react'
+import reactHooks from 'eslint-plugin-react-hooks'
+import { createRequire } from 'module'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+const require = createRequire(import.meta.url)
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+})
+
+export default [
+  {
+    ignores: ['src/generated/**/*', 'src/coverage/**/*', 'build/**/*', '**eslint.config.mjs'],
+  },
+  ...fixupConfigRules(
+    compat.extends(
+      'airbnb',
+      'airbnb/hooks',
+      'plugin:@typescript-eslint/recommended',
+      'plugin:react/recommended',
+      'plugin:react-hooks/recommended',
+      'prettier',
+      'plugin:jest/recommended',
+      'plugin:jest/style'
+    )
+  ),
```

Comment:
:wrench: I think it would be nice to only fixup plugins/configs where it is actually required to keep track what is still missing and what already supports the new config.

Reply:
I tried to move one by one but each one says `Cannot redefine plugin` except prettier showed me 7721 errors about Missing semicolons.
```
  ...compat.extends('prettier', 'plugin:jest/recommended', 'plugin:jest/style'),
  ...fixupConfigRules(
    compat.extends(
      'plugin:react/recommended',
      'plugin:react-hooks/recommended',
      'airbnb',
      'airbnb/hooks',
      'plugin:@typescript-eslint/recommended'
    )
  ),
```
It looks like the only way is to wrap it all with fixupConfigRules